lib/crawler_web/router.ex

```diff
@@ -55,4 +59,12 @@ defmodule CrawlerWeb.Router do
       post("/log_in", UserSessionController, :create)
     end
   end
+
+  scope "/api", CrawlerWeb, as: :api do
+    pipe_through [:api, :api_auth]
+
+    scope "/v1", Api.V1, as: :v1 do
+      post("/keyword/upload", KeywordController, :create)
```

Comment:
```suggestion
      post("/keyword", KeywordController, :create)
```

Reply:
Fixed in [9c72659](https://github.com/vnntsu/elixir-ic/pull/37/commits/9c72659be8ef3893874a455f0fb0f09da067cb86)